### ISSUE TITLE
Optionally store recurring transactions in Google tasks instead of calendar

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#245]

**Comment by klawdhfzasjhaa** [comment]
Would it be possible to handle it in MyExpenses completely like Financisto does it? As the default for new  devices is "Sync all to Google", it seems a bit inconvenient to share my financial schedule with Google.

**Comment by mtotschnig** [comment]
My Expenses by default creates a local calendar that is not linked to the Google account, and thus not synced to the cloud.
Handling recurrence completely in My Expenses is definitely possible. In fact it is already implemented for Blackberry, since the integration with the system calendar is not functional there. But it only allows the standard types like daily, weekly, monthly, yearly, with the possibility of custom recurrences that the platform calendar offers.

**Comment by klawdhfzasjhaa** [comment]
Ok, I see. I would rather not let financial data be saved into "calendar storage" at all. But that's only my personal view. [Financisto](https://github.com/dsolonenko/financisto2/search?utf8=✓&q=recurrence&type=) manages it internally, but I guess, it's not trivial.

**Comment by dutch602** [comment]
YES, YES, YES!!! Please remove external calendar dependence (e.g. Google Calendar) when creating templates and plans in My Expenses. I must redo each of mine manually whenever I upgrade to a new mobile phone after importing the backup/sync of My Expenses. The calendar events from the sync never work correctly on the new phone and my templates/plans must be recreated each time!

**Comment by mtotschnig** [comment]
https://github.com/mtotschnig/MyExpenses/issues/441#issuecomment-1987674567
https://github.com/mtotschnig/MyExpenses/issues/457#issuecomment-1987677942

**Comment by dutch602** [comment]
Thank you, I just responded to both of those comment threads. I think it
would help greatly if there was a way to completely eliminate the reliance
of an external calendar protocol (e.g. Google Calendar) and have My
Expenses manage everything internally within the app. I say this because
all calendar associations outside of the app do not carry over with a fresh
install via backup.
